tsconfig.json

```diff
@@ -6,19 +6,10 @@
     "jsx": "react",
     "skipLibCheck": true,
     "moduleResolution": "node",
-    "lib": [
-      "dom",
-      "es5",
-      "scripthost",
-      "es2015",
-      "es2015.promise",
-      "es2016.array.include",
-      "es2018",
-      "es2020"
-    ],
+    "lib": ["dom", "es5", "scripthost", "es2015", "es2015.promise", "es2016.array.include", "es2018", "es2020"],
     "resolveJsonModule": true,
     "noEmit": true,
     "target": "esnext"
   },
-  "include": ["src/**/*", "./setupTests.ts"]
+  "include": ["src/**/*", "./setupTests.ts", "tools/i18next-parser.config.js", "prettier.config.js"]
```

Comment:
I don't think these should be mentioned in `tsconfig` at all? They aren't even Typescript...